sync_ci/pkg/parser/regex_rules.go

```diff
@@ -0,0 +1,164 @@
+package parser
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/pingcap/log"
+	"io/ioutil"
+	"regexp"
+	"sync/atomic"
+	"time"
+	"unsafe"
+)
+
+var (
+	regexRules unsafe.Pointer
+	rulePath = "regex_rules.json"
+
+	timeRegexp = regexp.MustCompile(`\[20[0-9][0-9]-[0-9][0-9]-[0-9][0-9]T[0-9][0-9]:[0-9][0-9]:[0-9][0-9].[0-9][0-9][0-9]Z\]`)
+)
+
+type RegexRule struct {
```

Comment:
Is it necessary to add an alias of the matched string using rules and jobs, sometimes we use a key to represent the matched string.

Reply:
This string is used to move time from log.

Reply:
I mean the member of the struct `RegexRule`

Reply:
Let us do it later.